src/cli/subcommands/list.rs

```diff
@@ -132,43 +133,48 @@ fn list_projects(dir: StorageDir, list_config: &ListConfig) {
             ListMode::Nothing => print::print_projects(print::dynamic_rows(&projects,list_config)),
         }
     }
+    Ok(())
 }
 
 /// Command LIST --broken
-fn list_broken_projects(dir: StorageDir) {
-    let luigi = execute(setup::<Project>);
-    let invalid_files = execute(|| luigi.list_project_files(dir));
+fn list_broken_projects(dir: StorageDir) -> Result<()> {
```

Comment:
In all these functions it would be possible to use `let invalid_files = setup.<Project>()?.list_project_files(dir)?;`

If we do this we have no duplication with the included `asciii::storage`